src/functions-reference/higher-order_functions.Rmd

```diff
@@ -78,15 +81,30 @@ hybrid algorithm.
 Solves the algebraic system, given an initial guess, using the Powell
 hybrid algorithm with additional control parameters for the solver.
 
+*Note:* In future releases, the function `algebra_solver` will be deprecated
+and replaced with `algebra_solver_powell`.
```

Comment:
I would not make reference to a function which does not even exist at the moment. The Powell thing is not even available, right? So don't mention.

Reply:
We don't really justify why the Powell solver is simply called `algebra_solver`. As a reader I would find this odd. I think it's good to indicate our nomenclature is transitioning. This documentation is temporary anyways and will be updated once I put in a new PR with the renamed solver (and the variadic arguments).